### PR TITLE
Add missing overloads of hardware intrinsic

### PR DESCRIPTION
**Pull request by fiigii**
Match CoreCLR changes https://github.com/dotnet/coreclr/pull/20055 and https://github.com/dotnet/coreclr/pull/19420

@CarolEidt @tannergooding @eerhardt

### CODE REVIEW

**Comment by fiigii** [comment]
@dotnet-bot test this please

File: src/System.Runtime.Intrinsics/ref/System.Runtime.Intrinsics.cs [hunk 1]
```diff
@@ -483,7 +490,21 @@ public abstract class Avx2 : Avx
...
+        public static Vector256<ulong> BlendVariable(Vector256<ulong> left, Vector256<ulong> right, Vector256<ulong> mask) { throw null; }
         public static Vector128<T> BroadcastScalarToVector128<T>(Vector128<T> value) where T : struct { throw null; }
+        public static unsafe Vector128<byte> BroadcastScalarToVector128(byte* source) { throw null; }
+        public static unsafe Vector128<sbyte> BroadcastScalarToVector128(sbyte* source) { throw null; }
+        public static unsafe Vector128<short> BroadcastScalarToVector128(short* source) { throw null; }
+        public static unsafe Vector128<ushort> BroadcastScalarToVector128(ushort* source) { throw null; }
+        public static unsafe Vector128<int> BroadcastScalarToVector128(int* source) { throw null; }
+        public static unsafe Vector128<uint> BroadcastScalarToVector128(uint* source) { throw null; }
+        public static unsafe Vector128<long> BroadcastScalarToVector128(long* source) { throw null; }
+        public static unsafe Vector128<ulong> BroadcastScalarToVector128(ulong* source) { throw null; }
```
Comment (4creators) [inline src/System.Runtime.Intrinsics/ref/System.Runtime.Intrinsics.cs]: Why are we not having double* and single* overloads as well. Broadcast is type agnostic what matters is operand size.

File: src/System.Runtime.Intrinsics/ref/System.Runtime.Intrinsics.cs [hunk 2]
```diff
@@ -483,7 +490,21 @@ public abstract class Avx2 : Avx
...
         public static Vector256<sbyte> BlendVariable(Vector256<sbyte> left, Vector256<sbyte> right, Vector256<sbyte> mask) { throw null; }
         public static Vector256<byte> BlendVariable(Vector256<byte> left, Vector256<byte> right, Vector256<byte> mask) { throw null; }
+        public static Vector256<short> BlendVariable(Vector256<short> left, Vector256<short> right, Vector256<short> mask) { throw null; }
+        public static Vector256<ushort> BlendVariable(Vector256<ushort> left, Vector256<ushort> right, Vector256<ushort> mask) { throw null; }
+        public static Vector256<int> BlendVariable(Vector256<int> left, Vector256<int> right, Vector256<int> mask) { throw null; }
+        public static Vector256<uint> BlendVariable(Vector256<uint> left, Vector256<uint> right, Vector256<uint> mask) { throw null; }
+        public static Vector256<long> BlendVariable(Vector256<long> left, Vector256<long> right, Vector256<long> mask) { throw null; }
+        public static Vector256<ulong> BlendVariable(Vector256<ulong> left, Vector256<ulong> right, Vector256<ulong> mask) { throw null; }
```
Comment (4creators) [inline src/System.Runtime.Intrinsics/ref/System.Runtime.Intrinsics.cs]: Why are we not having double and single overloads as well. BlendVariable is type agnostic what matters is operand size.

**Comment by fiigii** [comment]
@tannergooding @jkotas The "Packaging All Configurations x64 Debug Build" seems to need more work than adding new API surface? Could you help?

**Comment by jkotas** [comment]
@morganbr I do not think we need to be running the API baseline checks for the netcoreappaot configuration. They cause extra busy work for folks adding new APIs. The situations like above are happening several times a week. The fix is always to just add the missing APIs to the baseline, so it is not really adding any value.

@fiigii Could you please try to fix this break by changing https://github.com/dotnet/corefx/blob/master/Directory.Build.props#L103 to `<RunApiCompat Condition="'$(TargetGroup)' != 'netcoreappaot'">true</RunApiCompat>` ?

**Comment by fiigii** [comment]
@jkotas Thank you so much! Will try.

**Comment by morganbr** [comment]
@jkotas, if we don't run it, I think that means we'll be breaking the netcoreappaot build regularly. Additionally, if the API in question isn't in netcoreappaot, it's probably also missing from uapaot, so something needs to be ported or baselined anyway. 

CC @ericstj

**Comment by fiigii** [comment]
@dotnet-bot test this please

**Pull request merged**